test/cypress/integration/demo-payment/mock-cards.cy.js

```diff
@@ -0,0 +1,36 @@
+const userStubs = require('../../stubs/user-stubs')
+const gatewayAccountStubs = require('../../stubs/gateway-account-stubs')
+const transactionStubs = require('../../stubs/transaction-stubs')
+
+const userExternalId = 'cd0fa54cf3b7408a80ae2f1b93e7c16e'
+const gatewayAccountId = '42'
+const gatewayAccountExternalId = 'a-valid-external-id'
+
+describe('Show Mock cards screen', () => {
+  beforeEach(() => {
+    cy.task('setupStubs', [
+      userStubs.getUserSuccess({ gatewayAccountId, userExternalId }),
+      gatewayAccountStubs.getGatewayAccountByExternalIdSuccess({
+        gatewayAccountId,
+        gatewayAccountExternalId,
+        paymentProvider: 'sandbox'
+      }),
+      transactionStubs.getTransactionsSummarySuccess()
+    ])
+  })
+
+  it('should load the mock cards page and show non stripe card', () => {
```

Comment:
Presumably we can have a version of this test that sets `paymentProvider` to `stripe` and checks we output the right card number for Stripe too?

Reply:
Agreed. Test to check correct card number is displayed for Stripe accounts now added. Note, unable to land directly on the Dashboard page for a Stripe account  as this route returned a ‘403 Forbidden’ status. Landing on the Stripe account’s Settings page and then navigating to the Dashboard pages worked but this probably needs refactoring.

Reply:
I think it’s fine — the amount of extra code to do the navigation is not huge and it’s not worth investing too much more time into this.